### ISSUE TITLE
Summer Update 2019

### ISSUE THREAD

**Issue by mathiasbynens** [tibiamaps/tibia-map-data#79]
The test server is now live: https://www.test.tibia.com/forum/?action=thread&threadid=271

[Kilmaresh (new continent)](https://www.test.tibia.com/forum/?action=thread&threadid=283)

**Comment by LeeKunTibia** [comment]
Let's get started! This is the first and largest batch, it should cover all of Kilmaresh:

[Kilmaresh.zip](https://github.com/tibiamaps/tibia-map-data/files/3364431/Kilmaresh.zip)

There are of course more areas to be mapped. Here's a list of changed/new areas, which are basically the locations of the Grave Danger Quest:

- Edron Graveyard
- Edron Crypt near the mountain (missing part of the Cube)
- Dark Cathedral
- Ghostlands
- Cormaya Cemetery
- Femor Hills
- Cobra Bastion NE of Ankrahmun
- Vengoth
- Somewhere in Darashia
- Somewhere in Thais Ancient Temple
- Somewhere at the Orclands
- Somewhere on the Ice Islands
- Last boss under Isle of the Kings

**Comment by LeeKunTibia** [comment]
So, here are almost all remaining files.

First, some parts of Kilmaresh were changed since the initial release, and the first batch was also missing the Ambassador's Mind map:
[Kilmaresh.zip](https://github.com/tibiamaps/tibia-map-data/files/3389203/Kilmaresh.zip)

The Cobra Bastion, east of Darama, including a small room under Muggy Plains that is accessed from the bastion:

[Cobra Bastion.zip](https://github.com/tibiamaps/tibia-map-data/files/3389204/Cobra.Bastion.zip)

The areas changed with Crypts for the Grave Danger Quest, plus the lever/boss rooms (around [here](https://tibiamaps.io/map#33479,31468,13:0)):

[Grave Danger.zip](https://github.com/tibiamaps/tibia-map-data/files/3389206/Grave.Danger.zip)

The following areas were changed:

- Inquisition headquarters:
https://tibiamaps.io/map#32331,32267,9:2

- Thais Ancient Temple:
https://tibiamaps.io/map#32355,32166,11:2

- Edron Graveyard and Crypt:
https://tibiamaps.io/map#33197,31687,7:1

- Dark Cathedral:
https://tibiamaps.io/map#32644,32392,8:1

- Ghostlands:
https://tibiamaps.io/map#32192,31821,8:2

- Cormaya Cemetery:
https://tibiamaps.io/map#33263,31990,7:2

- Femor Hills:
https://tibiamaps.io/map#32544,31851,6:1

- Vengoth:
https://tibiamaps.io/map#32930,31538,7:1

- Darashia Tumb:
https://tibiamaps.io/map#33289,32475,9:2

- Somewhere at the Orclands:
https://tibiamaps.io/map#32777,31829,7:2

- Folda:
https://tibiamaps.io/map#32007,31559,7:0

----

The following areas are still missing:
- Last boss entrance under Isle of the Kings
- Lass boss rooms

**Comment by LeeKunTibia** [comment]
I received the King Zelos maps from u/Moelnt on reddit! There is a small entrance under the Isle of the Kings:

https://tibiamaps.io/map#32172,31920,8:2

And the actual boss rooms, just south of the minibosses:

https://tibiamaps.io/map#33444,31555,13:0

Also, I noticed CipSoft removed a teleport and added a door to an area of the Kilmaresh Mountains, so I'm sending the updated files. This exact teleport was removed:

https://tibiamaps.io/map#33815,31651,9:2

[King Zelos and Mountain TP change.zip](https://github.com/tibiamaps/tibia-map-data/files/3449596/King.Zelos.and.Mountain.TP.change.zip)

This seems to be all _known_ new maps of the update. But since a new part of the Opticording Sphere Quest was apparently released, there may be more to come.
